sources/Outputs/include/Constants.h

```diff
@@ -37,27 +37,39 @@ hash(const std::string& str) {
- * 
- * @param generator the generator definition associated with the diagram name
+ *
+ * @param id the element id with the diagram name
  * @return The string filename of the diagram file
  */
-std::string diagramFilename(const dfl::algo::GeneratorDefinition& generator);
+std::string diagramFilename(const std::string& id);
 
-const std::string loadParId{"GenericRestorativeLoad"};                                ///< PAR id common to all loads
-const std::string diagramDirectorySuffix{"_Diagram"};                                 ///< Suffix for the diagram directory
-const std::string diagramMaxTableSuffix{"_tableqmax"};                                ///< Suffix for the table name for qmax in diagram file
-const std::string diagramMinTableSuffix{"_tableqmin"};                                ///< Suffix for the table name for qmin in diagram file
-const std::string signalNGeneratorParId{"signalNGenerator"};                          ///< PAR id for generators using signal N
-const std::string signalNGeneratorFixedPParId{"signalNGeneratorFixedP"};              ///< PAR id for generators using signal N with fixed P
-const std::string propSignalNGeneratorParId{"propSignalNGenerator"};                  ///< PAR id for generators using prop signal N
-const std::string propSignalNGeneratorFixedPParId{"propSignalNGeneratorFixedP"};      ///< PAR id for generators using prop signal N with fixed P
-const std::string remoteSignalNGeneratorParId{"remoteSignalNGenerator"};              ///< PAR id for generators using remote signal N
-const std::string remoteSignalNGeneratorFixedPParId{"remoteSignalNGeneratorFixedP"};  ///< PAR id for generators using remote signal N with fixed P
-const std::string xmlEncoding{"UTF-8"};                                               ///< Default encoding for XML outputs files
+const std::string loadParId{"GenericRestorativeLoad"};                            ///< PAR id common to all loads
+const std::string diagramDirectorySuffix{"_Diagram"};                             ///< Suffix for the diagram directory
+const std::string diagramMaxTableSuffix{"_tableqmax"};                            ///< Suffix for the table name for qmax in diagram file
+const std::string diagramMinTableSuffix{"_tableqmin"};                            ///< Suffix for the table name for qmin in diagram file
+const std::string signalNGeneratorParId{"signalNGenerator"};                      ///< PAR id for generators using signal N
+const std::string signalNGeneratorFixedPParId{"signalNGeneratorFixedP"};          ///< PAR id for generators using signal N with fixed P
+const std::string propSignalNGeneratorParId{"propSignalNGenerator"};              ///< PAR id for generators using prop signal N
+const std::string propSignalNGeneratorFixedPParId{"propSignalNGeneratorFixedP"};  ///< PAR id for generators using prop signal N with fixed P
+const std::string remoteSignalNParId{"remoteSignalN"};                            ///< PAR id for using remote signal N
```

Comment:
Why have you remove generator here ?

Reply:
Because it is also used for HVDC lines

Reply:
Well then the name might be confusing as SignalN is only used for generators and not HVDC.
remoteVControl or remoteSignalNQ could be more appropriate I think

Reply:
change to remoteVControl. Generator word re-added for fixedP version